class-13.md

```diff
@@ -0,0 +1,138 @@
+# Intro to Blockchain Development
+
+### Class 13 - More Hyperledger Examples 
+
+We got started with private blockchains last week, exploring them through Hyperledger Composer and Fabric. Now we'll go a bit more in depth and look onto some specific code examples further.
+
+
+
+Reading & Videos:
+
+- [Composer Sample Networks](https://github.com/hyperledger/composer-sample-networks)* - This is a set of composer examples from the hyperledger organization. Try running one to see how it works, and get some ideas for how to implement solutions to problems in a permissioned blockchain.
```

Comment:
~Try running one to see how it works, and get some ideas for how to implement solutions to problems in a permissioned blockchain.~

Try running any of the examples to help you understand how a permissioned blockchain works.